ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+<!--
+
+Thank you for taking time to open an issue.
+Please take time to read the instructions, even if you've read it before, as things may have changed in the recent past.
+
+There are three main reasons you may want to open an issue here:
+
+1. a bug report
+2. a feature request
+3. operational issues
+
+#################################################################################
+IN ALL OF THESE CASES, PLEASE SEARCH FOR RELATED ISSUES BEFORE OPENING A NEW ONE.
```

Comment:
Given that this is shared for all cases I'd go with a combination template for both Bugs/ Feature requests and mention this on the first checklist, simplyfing the text to: 

- [ ] this is a bug / feature request (not an operational issue) 
- [ ] I have checked https://github.com/travis-ci/travis-ci/issues for potentially related issues
- [ ] I have read the docs, in particular, https://docs.travis-ci.com/user/common-build-problems/